api/jobs_test.go

```diff
@@ -367,12 +376,19 @@ func ExpectedJob(id string,
 	state string,
 	totalSearchDocuments int,
 	totalInsertedSearchDocuments int) models.Job {
+	cfg, err := config.Get()
+	if err != nil {
+		err = errors.New("unable to retrieve service configuration")
```

Comment:
Again here the causing error should be wrapped.. But also the error isn't being returned? Looks like it should be otherwise there will be panics/unexpected behaviour trying to work with a nil config value.

Reply:
makes sense - done